### PR TITLE
Dataset

### PR DESCRIPTION
**Pull request by chiahsienlin**

### CODE REVIEW

File: doc/proposal/proposal.tex [hunk 1]
```diff
@@ -63,37 +63,27 @@ \section{Proposed Methods}
...
-	\item{Source URL: https://data.cityofnewyork.us/Housing-Development
-	/DOB-Job-Application-Filings/ic3t-wcy2}
-	\item{Data Property: categorical dataset}
-	\item{Rows: 5,260,437, Columns: 89}
-	\item{Size: 2.88G}
-	\item{Missing Value: There are about 36 out of 89 columns with null value which is about 40\%.}
-	\item{Reason to choose the data: In this dataset, there are 89 columns which satisfies the categorical property we need in this project.}
+	\item{311 Service Requests}\cite{nycopendata1} This is a dataset containing all 311 Service Requests from 2010 to present. In this dataset, most of the data is consisted of alphabetical words which is one of the data property we are looking for. Besides, the column named "Resolution Description" contains the description about the service status notes.
+	\item{DOB Permit Issuance}\cite{nycopendata2} The data is a list of permits issued by Department of Building for particular borough, street, house in NYC. In this dataset, most of the data is consisted of numbers such as Bin number, Job number columns and so on. We would like to test data scaling and outlier detection on this dataset.
```
Comment (plliao) [inline doc/proposal/proposal.tex]: Job number columns or Job number?

Comment (chiahsienlin) [reply doc/proposal/proposal.tex]: ..... such as columns containing Bin numbers, job numbers and so on.

Comment (plliao) [reply doc/proposal/proposal.tex]: 改一下拔, 改完要merge惹

Comment (chiahsienlin) [reply doc/proposal/proposal.tex]: ok

**Pull request merged**